src/main/java/com/rultor/agents/ecs/StartsECS.java

```diff
@@ -41,29 +41,32 @@
 import org.xembly.Directives;
 
 /**
- * Starts Amazon EC2 instance.
+ * Starts Amazon ECS container, that runs Docker + SSHD on ECS. Container
+ * is configured in privileged mode. Configuration is stored in S3.
+ * See details here:
+ * @link http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html#ecs-config-s3
  * @author Yuriy Alevohin (alevohin@mail.ru)
  * @version $Id$
  * @since 2.0
- * @todo #629 Implement com.rultor.agents.ec2.StopsEC2 agent. It must
- *  stopped EC2 on-demand instance if it was started at StartsEC2 agent.
- *  StopsEC2 must use instance id from /talk/ec2/[@id] to stop it.
+ * @todo #629 Implement com.rultor.agents.ecs.StopsECS agent. It must
+ *  stop ECS on-demand container if it was started at StartsECS agent.
+ *  StopsECS must use container ARN from /talk/ecs/[@id] to stop it.
  * @todo #629 RegistersShell must register SSH params "host", "port",
- *  "login", "key" for ec2 on-demand instance, if this one was successfully
+ *  "login", "key" for ecs on-demand container, if this one was successfully
  *  started. Successfully start means that these parameters exist in
- *  /talk/ec2
- * @todo #629 Add new instance creation classes for StartsEC2 and StopsEC2
- *  to com.rultor.agents.Agents. StartsEC2 must be invoked before
- *  RegistersShell agent. StopsEC2 must be invoked after RemovesShell agent.
- * @todo #629 Write documentation for configuring ec2 via .rultor.yml at
+ *  /talk/ecs
+ * @todo #629 Add new container creation classes for StartsECS and StopsECS
+ *  to com.rultor.agents.Agents. StartsECS must be invoked before
+ *  RegistersShell agent. StopsECS must be invoked after RemovesShell agent.
+ * @todo #629 Write documentation for configuring ecs via .rultor.yml at
```

Comment:
@xupyprmv you know what this puzzle is so ambiguous (it was before and it is now), let's just put this instead:

`Implement the ability to use Rultor with any AWS account, by providing the credentials to it via the encrypted assets and a respective configuration setting in the .rultor.yml. Document the implementation properly in the Rultor documentation.`